tests/test_docker.py

```diff
@@ -88,6 +88,69 @@ def test_instance_activate_need_pull_image(agent, responses):
 
 
 @if_docker
+def test_docker_client_pull():
```

Comment:
Do you intend to keep this test or is it a temporary one that you'll throw away?